Sources/Broadcast/BroadcastViewModifier.swift

```diff
@@ -0,0 +1,49 @@
+import SwiftUI
+
+extension Warp {
+    public struct BroadcastViewModifier: ViewModifier {
+        let text: String
+        let edge: BroadcastEdge
+        private let horizontalPadding: Double = 16
```

Comment:
Is this a static constant?
If not then it can not be injected now.

Reply:
Yes I would say padding we control and users should not be able to change it.